### DISCUSSION TITLE
Surface Book 3 Nvidia Driver

### DISCUSSION THREAD

**Discussion by whoIsLosti** [linux-surface/linux-surface#1212] [Q&A]
Hey, after successfully installing the Linux Surface drivers for my Surface Book 3, I now want to install the appropriate drivers for my NVIDIA 1650 (Max-Q). You can find the details about my operating system in the attached picture.

My question now is where and how I can get the right drivers for my graphics card? Are there special kernel modules for the driver of the Surface Book on this git or do I have to get them externally? Are there any special requirements because my graphics card is an Optimus-card?
I have already tried the rpm-Fusion drivers, but without success.

I am grateful for any help!

![Screenshot_Neofetch](https://github.com/linux-surface/linux-surface/assets/124056428/556ed87a-ad2a-4ab2-9c04-45e4723f6f25)

**Comment by qzed** [comment]
You generally have to install nvidia drivers separately since they're not included in the kernel. For Fedora, this is documented at https://rpmfusion.org/Howto/NVIDIA. Or steps 1 to 7 (inclusively) at https://docs.fedoraproject.org/en-US/quick-docs/how-to-set-nvidia-as-primary-gpu-on-optimus-based-laptops/ (I recommend you don't set it as primary GPU, so don't run steps 8 and after).
